mlflow/pyfunc/scoring_server.py

```diff
@@ -97,6 +106,19 @@ def init(model):
     """
     app = flask.Flask(__name__)
 
+    eprint("**IMPORTANT UPDATE**: Starting in MLflow 0.9.0, requests received with a"
```

Comment:
Maybe we could print this warning the first time we receive an `application/json` Content-Type? That way people can "fix" the warning, and are not bothered if they're just using CSV or something already.